src/Header.jsx

```diff
@@ -28,7 +28,7 @@ function SiteHeader({ intl }) {
     },
   ];
 
-  const userMenu = [
+  const userMenu = authenticatedUser === null ? [] : [
```

Comment:
userMenu uses authenticatedUser.username below this - I'm just emptying it out rather than messing with menu items that won't get used.